pipeline/config-dev.groovy

```diff
@@ -178,6 +179,32 @@ app {
                             'ENVIRONMENT_NAME':"${app.deployment.env.name}",
                             'API_URL': "https://${vars.modules.'mds-nginx'.HOST_CORE}${vars.modules.'mds-nginx'.PATH}/api",
                     ]
+                ],
+                [
+                    'file':'microservices/nris_api/openshift/_python36_oracle.dc.json',
+                    'params':[
+                            'NAME':"mds-nris-backend",
+                            'SUFFIX': "${vars.deployment.suffix}",
+                            'VERSION':"${app.deployment.version}",
+                            'CPU_REQUEST':"${vars.resources.python.cpu_request}",
+                            'CPU_LIMIT':"${vars.resources.python.cpu_limit}",
+                            'MEMORY_REQUEST':"${vars.resources.python.memory_request}",
```

Comment:
We may want to tweak the memory/cpu request and pod resources when this reaches test and prod.
Since its much more lightweight it probably can get away with less resources than our API

Reply:
Probably true, good point, not sure i'm worried about that level in the PR, let me know if you want those changes now.

Reply:
Eh i mean dev is good. Can deal with that later when it gets to test/prod 💯